nflow-engine/src/main/java/io/nflow/engine/internal/dao/TableMetadataChecker.java

```diff
@@ -68,7 +70,7 @@ public void ensureCopyingPossible(String sourceTable, String destinationTable) {
     @Override
     public Map<String, ColumnMetadata> extractData(ResultSet rs) throws SQLException {
       ResultSetMetaData metadata = rs.getMetaData();
-      Map<String, ColumnMetadata> metadataMap = new LinkedHashMap<>();
+      Map<String, ColumnMetadata> metadataMap = new LinkedHashMap<>(metadata.getColumnCount(), 1.0f);
```

Comment:
Why add an explicit fill factor? And the closer to `1` the value is the more likely it is that the map will become linked list. We're not trying to save any memory here so I think the deafult `0.75` is good here.

Reply:
Updated code to use double initial size and default load factor (double size to avoid rehashing)